README.md

```diff
@@ -35,13 +35,13 @@ Goal: SDK size <= 1 MB and combined resources (object detection + regression mod
 
-Drishti is a [CMake](https://github.com/kitware/CMake) based project that uses the [Hunter](https://github.com/ruslo/hunter) package manager to download and build project dependencies from source as needed.  Hunter contains [detailed documentation](https://docs.hunter.sh/en/latest), but a few high level notes are provided here to help orient first time users.  In practice, some working knowledge of CMake may also be required.  Hunter itself is written in CMake, and is installed as part of the build process from a single `HunterGate()` macro at the top of the root `CMakeLists.txt` file (typically `cmake/Hunter/HunterGate.cmake`) (you don't have to build or install it).  Each CMake dependency's `find_package(FOO)` call that is paired with a `hunter_add_package(FOO CONFIG REQUIRED)` will be managed by Hunter.  Additional configuration (i.e., custom version and/or CMake arguments) can be performed for each package with an optional `hunter_config(FOO VERSION 1.0.0 CMAKE_ARGS ${FOO_CMAKE_ARGS}))` call in an optional local config file that is specified in the `HunterGate()` call.  In most cases, the only system requirement for building a Hunter project is a recent CMake with CURL support and a working compiler correpsonding to the operative toolchain. 
+Drishti is a [CMake](https://github.com/kitware/CMake) based project that uses the [Hunter](https://github.com/ruslo/hunter) package manager to download and build project dependencies from source as needed.  Hunter contains [detailed documentation](https://docs.hunter.sh/en/latest), but a few high level notes are provided here to help orient first time users.  In practice, some working knowledge of CMake may also be required.  Hunter itself is written in CMake, and is installed as part of the build process from a single `HunterGate()` macro at the top of the root `CMakeLists.txt` file (typically `cmake/Hunter/HunterGate.cmake`) (you don't have to build or install it).  Each CMake dependency's `find_package(FOO)` call that is paired with a `hunter_add_package(FOO CONFIG REQUIRED)` will be managed by Hunter.  Additional configuration (i.e., custom version and/or CMake arguments) can be performed for each package with an optional `hunter_config(FOO VERSION 1.0.0 CMAKE_ARGS ${FOO_CMAKE_ARGS}))` call in an optional local config file that is specified in the `HunterGate()` call.  In most cases, the only system requirement for building a Hunter project is a recent CMake with CURL support and a working compiler correpsonding to the operative toolchain.  Hunter will maintain all dependencies in a [versioned](https://docs.hunter.sh/en/latest/overview/customization.html) local [cache](https://docs.hunter.sh/en/latest/overview/shareable.html) by default (typically `${HOME}/.hunter`) where they can be reused in subsequent builds and shared between different projects.  They can also be stored in a server side [binary cache](https://docs.hunter.sh/en/latest/overview/binaries.html).
```

Comment:
> Additional configuration (i.e., custom version and/or CMake arguments) can be performed for each package with an optional `hunter_config(FOO VERSION 1.0.0 CMAKE_ARGS ${FOO_CMAKE_ARGS}))` call in an optional local config file that is specified in the `HunterGate()` call

Not sure why user may need to know this.

> In most cases, the only system requirement for building a Hunter project is a recent CMake with CURL support and a working compiler correpsonding to the operative toolchain

Info is here:
* https://docs.hunter.sh/en/latest/contributing.html#reporting-bugs

Probably should be moved to "Quick start: Tools".

> They can also be stored in a server side [binary cache](https://docs.hunter.sh/en/latest/overview/binaries.html).

Not very informative. How it can be helpful for the first time user?

Reply:
> Not sure why user may need to know this.

I think that makes sense.  That is probably more critical for creating a new project.  I'll cut this (in line w/ preference for bare bones description that doesn't duplicate Hunter docs too much).

> Info is here: https://docs.hunter.sh/en/latest/contributing.html#reporting-bugs

I'll link this.  I think it is important for the "setup" notes.  I believe the default Ubuntu 16.04 doesn't have CURL support, for example.  (I'll incorporate feedback and update via direct README.md commit after the merge.)

> Not very informative. How it can be helpful for the first time user?

In the toolchain discussion, the CI builds + cache are mentioned.  I think it is important to introduce the idea of a binary cache so the user understands the pre-built packages are downloaded from a server.  Maybe a little more information could help.

Reply:
> I believe the default Ubuntu 16.04 doesn't have CURL support, for example

I'm pretty sure it has. The CURL support issue (OpenSSL support to be precise) usually about manually compiled CMake.

> In the toolchain discussion, the CI builds + cache are mentioned

Okay, I see. However not all of them are "stable", e.g. `xcode-hid-sections`.